data/org.freedesktop.impl.portal.Usb.xml

```diff
@@ -0,0 +1,81 @@
+<?xml version="1.0"?>
+<!--
+ Copyright (C) 2023 GNOME Foundation Inc.
+
+ This library is free software; you can redistribute it and/or
+ modify it under the terms of the GNU Lesser General Public
+ License as published by the Free Software Foundation; either
+ version 2 of the License, or (at your option) any later version.
+
+ This library is distributed in the hope that it will be useful,
+ but WITHOUT ANY WARRANTY; without even the implied warranty of
+ MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU
+ Lesser General Public License for more details.
+
+ You should have received a copy of the GNU Lesser General Public
+ License along with this library. If not, see <http://www.gnu.org/licenses/>.
+
+ Author: Georges Basile Stavracas Neto <georges.stavracas@gmail.com>
+-->
+
+<node name="/" xmlns:doc="http://www.freedesktop.org/dbus/1.0/doc.dtd">
+  <!--
+      org.freedesktop.impl.portal.Usb:
+      @short_description: USB portal backend interface
+
+      This portal lets applications register global shortcuts so they can
```

Comment:
The description seems wrong, this has afaik nothing to do with global shortcut

Reply:
oops. I missed that too. Will address this.

Reply:
See https://github.com/flatpak/xdg-desktop-portal/pull/1529

Reply:
not sure about the annotation. Will followup too.

Reply:
added the annotations to the same PR.